### PR TITLE
Parse and return restic physical size

### PR DESCRIPTION
**Pull request by redgoat650**
## Change Overview

Parse backup actions for the physical size added to a repository, return as part of the backup and copy call.

## Pull request type

Please check the type of change your PR introduces:
- [x] :sunflower: Feature

## Test Plan
TestGetSnapshotStatsFromBackupLog
- [ ] :muscle: Manual
- [x] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/restic/restic.go [hunk 1]
```diff
@@ -342,26 +342,38 @@ func SnapshotIDFromBackupLog(output string) string {
...
 
 // SnapshotStatsFromBackupLog gets the Snapshot file count and size from Backup Command log
-func SnapshotStatsFromBackupLog(output string) (fileCount string, backupSize string) {
+func SnapshotStatsFromBackupLog(output string) (fileCount string, backupSize string, phySize string) {
 	if output == "" {
-		return "", ""
+		return "", "", ""
 	}
 	logs := regexp.MustCompile("[\n]").Split(output, -1)
 	// Log should contain "processed %d files, %.3f [Xi]B in mm:ss"
```
Comment (SupriyaKasten) [inline pkg/restic/restic.go]: nit: please also add comment for logs containing `Added to the repo....`

**Comment by SupriyaKasten** [comment]
LGTM 👍 🚀

File: pkg/restic/restic.go (hunk 1 above)
Comment (redgoat650) [reply pkg/restic/restic.go]: done

**Comment by redgoat650** [comment]
Thanks @SupriyaKasten and @pavannd1 !

**Pull request merged**